root/scripts/editRecipe.js

```diff
@@ -98,9 +98,7 @@ async function fillRecipePage(recipeId) {
   }
 
   for (let k = 1; k <= recipe.steps.length; k += 1) {
-    if (k > 5) {
-      addStep();
-    }
+    addStep();
```

Comment:
Yeah not too sure the original intuition with the (k > 5) condition, but your changes seem to fix the issue and are minimal. Good job! Just address some of my other comments and you should be good to go.